front/lib/models/workspace.ts

```diff
@@ -119,6 +119,7 @@ Workspace.hasMany(Membership, {
   foreignKey: { allowNull: false },
   onDelete: "CASCADE",
 });
+Membership.belongsTo(Workspace);
```

Comment:
What's the purpose?

Reply:
It allows to do a sequelize query such as `Membership.find(includes: [Workspace])`
Otherwise sequelize is not aware of the connection and answers "workspace has no membership"
I needed it initially, now I don't since I go through getMembers

IMHO good to declare relationships anyways (took me 15mns to find out what's going on so that's time saved for the next one who might need it) but not strongly held